props/saber_shtok_buttons.h

```diff
@@ -153,6 +240,69 @@ SaberShtokButtons() : PropBase() {}
         }
         return true;
 
+// Battle Mode
+
+#ifndef FETT263_BATTLE_MODE_ALWAYS_ON
+      case EVENTID(BUTTON_NONE, EVENT_SWING, MODE_OFF | BUTTON_POWER):
+        if (!battle_mode_) {
+          battle_mode_ = true;
+	  if(millis() > 3000) { 
```

Comment:
indentation looks wrong here and in other places, although that might because of github.
I can always fix it after it's checked in, but take a look.

Reply:
I'm not sure about all this coding stuff, Fernando helped me to make this file and I tested it, it works fine :)

Reply:
I can fix the indentation, but the defines have to be fixed.